### PR TITLE
PYIC-7494 Use vocab classes for CIMIT

### PR DESCRIPTION
**Pull request by Joe-Edwards-GDS**
### What changed

Use the vocab classes for CIs and Mitigations

### Why did it change

No need to use our own copies which may get out of date

### Issue tracking

- [PYIC-7494](https://govukverify.atlassian.net/browse/PYIC-7494)

[PYIC-7494]: https://govukverify.atlassian.net/browse/PYIC-7494?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/build-cri-oauth-request/build.gradle [hunk 1]
```diff
@@ -7,7 +7,6 @@ plugins {
 
 dependencies {
 	implementation libs.bundles.awsLambda,
-			libs.diVocab,
```
Comment (Joe-Edwards-GDS) [inline lambdas/build-cri-oauth-request/build.gradle:10]: (added as an api dependency in `common-services`, so this is now available anywhere that uses the common lib)

File: libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CimitService.java [hunk 2]
```diff
@@ -109,23 +107,41 @@ public void submitMitigatingVcList(
...
+    public List<ContraIndicator> getContraIndicators(
             String userId, String govukSigninJourneyId, String ipAddress)
             throws CiRetrievalException {
         var vc = getContraIndicatorsVc(userId, govukSigninJourneyId, ipAddress);
 
         return getContraIndicators(vc);
     }
 
-    public ContraIndicators getContraIndicators(VerifiableCredential vc)
+    public List<ContraIndicator> getContraIndicators(VerifiableCredential vc)
```
Comment (Joe-Edwards-GDS) [inline libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CimitService.java:118]: This is the main change - returning the CIs directly from the VC object

File: libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/CimitUtilityService.java [hunk 3]
```diff
@@ -22,18 +25,63 @@ public CimitUtilityService(ConfigService configService) {
         this.configService = configService;
     }
 
+    public int getContraIndicatorScore(List<ContraIndicator> contraIndicators)
```
Comment (Joe-Edwards-GDS) [inline libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/CimitUtilityService.java:28]: This was only used in `CimitUtilityService` anyway, so it makes sense to move here and keep the CIs as pure data classes.

**Review by Wynndow — APPROVED** [review]
Nice!

**Pull request merged**